src/app/src/components/CellElement.jsx

```diff
@@ -126,144 +68,11 @@ export default class CellElement extends Component {
-                    confirmRejectDialogStates.reject
-                }
-                aria-labelledby="alert-dialog-title"
-                aria-describedby="alert-dialog-description"
-            >
-                <DialogTitle id="alert-dialog-title">
-                    Reject this potential facility match?
-                </DialogTitle>
-                <DialogContent id="alert-dialog-description">
-                    <h3>
-                        This action will reject the facility as a potential
-                        match.
-                    </h3>
-                    <p>
-                        <strong>Potential match:</strong>
-                    </p>
-                    <ul>
-                        <li>name: {item.matchName}</li>
-                        <li>address: {item.matchAddress}</li>
-                    </ul>
-                    <p>
-                        <strong>List item:</strong>
-                    </p>
-                    <ul>
-                        <li>name: {item.itemName}</li>
-                        <li>address: {item.itemAddress}</li>
-                    </ul>
-                    <p>
-                        If no other potential matches remain, this will create a
-                        new facility.
-                    </p>
-                </DialogContent>
-                <DialogActions>
-                    <Button
-                        variant="outlined"
-                        color="secondary"
-                        onClick={this.closeDialog}
-                    >
-                        No, do not reject
-                    </Button>
-                    <Button
-                        variant="contained"
-                        color="secondary"
-                        onClick={this.rejectFacilityMatch}
-                    >
-                        Yes, reject
-                    </Button>
-                </DialogActions>
-            </Dialog>
-        );
-
-        return (
-            <div
-                key={item.id}
-                style={confirmRejectMatchRowStyles.cellRowStyles}
-            >
-                <div style={confirmRejectMatchRowStyles.cellActionStyles}>
-                    <Button
-                        variant="contained"
-                        color="primary"
-                        onClick={this.openConfirmDialog}
-                        disabled={fetching}
-                    >
-                        Confirm
-                    </Button>
-                    <Button
-                        variant="contained"
-                        color="secondary"
-                        onClick={this.openRejectDialog}
-                        disabled={fetching}
-                    >
-                        Reject
-                    </Button>
-                    {confirmDialog}
-                    {rejectDialog}
-                </div>
-            </div>
-        );
+        return null;
```

Comment:
Do we still need CellElement.jsx?

Reply:
Yes, we still use it since it has multiple responsibilities (which is not right imho)

Reply:
Please add TODO there, so we need to refactor it in nearest future.

Reply:
@VadimKovalenkoSNF

Reply:
Done